Sources/SwiftInspectorVisitors/Tests/PropertyVisitorSpec.swift

```diff
@@ -117,7 +117,7 @@ final class PropertyVisitorSpec: QuickSpec {
               name: "thing",
               typeDescription: .simple(name: "String"),
               modifiers: [.public, .static],
-              paradigm: .definedVariable("\"Hello, World\""))
+              paradigm: .definedVariable("= \"Hello, World\""))
```

Comment:
I originally planned to remove this though when I wrote the code it felt more logical to include it since it is a part of the initializer syntax node.

Reply:
I don't love including this constant prefix – I'm having trouble seeing how it's valuable.

I don't think it's problematic, but feels like `initializerDescription` already tells us that we're looking at the contents of an initializer so we don't need to repeat the `= `.

Maybe I'm missing something though. What led you to add this data?

Reply:
When I was writing [this](https://github.com/fdiaz/SwiftInspector/blob/57ddcd73a5e7d684fd310c23bcf11012a24e4ca9/Sources/SwiftInspectorVisitors/PropertyVisitor.swift#L160-L164) method I found that the code to remove the syntax node with the `=` was a bit messy. Basically what I had was something like this:

```swift
return initializerClauseSyntaxes.first?.children?.last?.description
```

I had found that the children of `InitializerClauseSyntax` _seemed_ to be two nodes, the first of which contained the `=`. I was then starting to assert that `InitializerClauseSyntax` only contained two nodes, and it began to feel like brittle code.

I began to feel like I was making assumptions that may not be valuable for everyone using this library, and it felt like there was simplicity in `initializerDescription` containing the literal source of the `InitializerClauseSyntax` object.

As I thought further, one nice thing about the way we have it now is that a consumer of this visitor could theoretically reparse `initializerDescription` with SwiftSyntax at a higher layer and get back an `InitializerClauseSyntax`. That felt like a nice contract.

I could be convinced to remove the `=` (which we will do in our higher layer anyway) though I had trouble convincing myself that the added code here was adding a lot of value compared to the complexity, though I agree the tests are a bit annoying.

Reply:
Got it! Okay I didn't realize that removing the `= ` added so much complexity and potential for failure points – I'm now entirely onboard with this approach.

Reply:
Unfortunately reparsing the description won't give back an `InitializerClauseSyntax`. That being said I still think we stay simple until the range of use cases become more apparent.
![image](https://user-images.githubusercontent.com/1791049/133539171-83d2ccf8-65a9-4837-843e-9e049b62c39f.png)

Reply:
@bachand did we try something like `.withEqual(nil)` on the node?

```swift
/// Returns a copy of the receiver with its `equal` replaced.
  /// - param newChild: The new `equal` to replace the node's
  ///                   current `equal`, if present.
  public func withEqual(
    _ newChild: TokenSyntax?) -> InitializerClauseSyntax {
    let raw = newChild?.raw ?? RawSyntax.missingToken(TokenKind.equal)
    let newData = data.replacingChild(raw, at: Cursor.equal)
    return InitializerClauseSyntax(newData)
  }
```

Reply:
Yup, that works https://github.com/fdiaz/SwiftInspector/pull/111

Reply:
I liked where we landed in that PR :)